charts/cluster-templates/templates/cluster.yaml

```diff
@@ -46,7 +46,10 @@ spec:
   {{- end }}
   # redeploySystemAgentGeneration:
   rkeConfig:
-    # additionalManifest:
+    {{- with $.Values.cluster.config.additionalManifests }}
```

Comment:
hey @atoy3731, curious the difference in using the `with` vs `if`?

Reply:
Essentially just a "shorthand" way to reference something from the values.yaml and implement it only if it exists and has a value. If you do a `helm create foobar` and check out the `templates/deployment.yaml` that gets created, you'll see a bunch of references around what it is doing.